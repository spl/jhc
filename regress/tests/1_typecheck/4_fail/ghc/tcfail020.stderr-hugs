ERROR "tcfail020.hs":10 - Cannot build superclass instance
*** Instance            : B [a]
*** Context supplied    : A a
*** Required superclass : A [a]
