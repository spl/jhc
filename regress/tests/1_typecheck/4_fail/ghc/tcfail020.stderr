
tcfail020.hs:10:10:
    Could not deduce (A [a])
      arising from the superclasses of an instance declaration
    from the context (A a)
      bound by the instance declaration at tcfail020.hs:10:10-23
    Possible fix:
      add (A [a]) to the context of the instance declaration
      or add an instance declaration for (A [a])
    In the instance declaration for `B [a]'
